packages/round-manager/.eslintrc.js

```diff
@@ -20,6 +20,14 @@ module.exports = {
       rules: {
         "@typescript-eslint/no-empty-function": "off",
         "@typescript-eslint/no-non-null-assertion": "off",
+        "@typescript-eslint/no-unused-vars": [
+          "warn",
+          {
+            varsIgnorePattern: "_",
```

Comment:
`"_"` matches _any_ string containing `_` , so the rule will ignore all unused variable names containing `_` like `_thing`, `some_thing`, etc

`"^_$"` will match exactly `_` only

Reply:
resolved in [8237eaa](https://github.com/gitcoinco/grants-round/pull/456/commits/8237eaae9760d2d6da5f60c9b748f2c91fa1ab36)